upload/admin/model/customer/customer.php

```diff
@@ -220,44 +220,44 @@ public function getAddresses($customer_id) {
 	}
 
 	public function getTotalCustomers($data = []) {
-		$sql = "SELECT COUNT(*) AS total FROM " . DB_PREFIX . "customer c";
+		$sql = "SELECT COUNT(*) AS total FROM `" . DB_PREFIX . "customer` c";
 
 		if (!empty($data['filter_affiliate'])) {
-			$sql .= " LEFT JOIN " . DB_PREFIX . "customer_affiliate ca ON (ca.customer_id = c.customer_id)";
+			$sql .= " LEFT JOIN `" . DB_PREFIX . "customer_affiliate` ca ON (ca.`customer_id` = c.`customer_id`)";
 		}
 
 		$implode = [];
 
 		if (!empty($data['filter_name'])) {
-			$implode[] = "CONCAT(c.firstname, ' ', c.lastname) LIKE '%" . $this->db->escape((string)$data['filter_name']) . "%'";
+			$implode[] = "CONCAT(c.`firstname`, ' ', c.`lastname`) LIKE '%" . $this->db->escape((string)$data['filter_name']) . "%'";
 		}
 
 		if (!empty($data['filter_email'])) {
-			$implode[] = "c.email LIKE '" . $this->db->escape((string)$data['filter_email']) . "%'";
+			$implode[] = "c.`email` LIKE '" . $this->db->escape((string)$data['filter_email']) . "%'";
 		}
 
 		if (isset($data['filter_newsletter']) && !is_null($data['filter_newsletter'])) {
-			$implode[] = "c.newsletter = '" . (int)$data['filter_newsletter'] . "'";
+			$implode[] = "c.`newsletter` = '" . (int)$data['filter_newsletter'] . "'";
 		}
 
 		if (!empty($data['filter_customer_group_id'])) {
-			$implode[] = "c.customer_group_id = '" . (int)$data['filter_customer_group_id'] . "'";
+			$implode[] = "c.`customer_group_id` = '" . (int)$data['filter_customer_group_id'] . "'";
 		}
 
 		if (!empty($data['filter_affiliate'])) {
-			$implode[] = "ca.status = '" . (int)$data['filter_affiliate'] . "'";
+			$implode[] = "ca.`status` = '" . (int)$data['filter_affiliate'] . "'";
 		}
 
 		if (!empty($data['filter_ip'])) {
-			$implode[] = "c.customer_id IN (SELECT customer_id FROM " . DB_PREFIX . "customer_ip WHERE ip = '" . $this->db->escape((string)$data['filter_ip']) . "')";
+			$implode[] = "c.`customer_id` IN (SELECT `customer_id` FROM " . DB_PREFIX . "customer_ip WHERE `ip` = '" . $this->db->escape((string)$data['filter_ip']) . "')";
```

Comment:
Wondering if this line shouldn't be:

$implode[] = "c.`customer_id` IN (SELECT ca.`customer_id` FROM " . DB_PREFIX . "customer_ip WHERE ca.`ip` = '" . $this->db->escape((string)$data['filter_ip']) . "' AND ca.`customer_id` = c.`customer_id`)";